spec/mailers/admin_mailer_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe AdminMailer, type: :mailer do
+  describe "notify" do
+    let(:user) { FactoryBot.create(:user, role: :admin) }
+    let(:mail) { AdminMailer.admin_notification(user) }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq("New user was registered!")
```

Comment:
Use i18n

Reply:
There is no need, because subject was only in English